src/Polished/Polished_Color.re

```diff
@@ -5,6 +5,10 @@ let lighten = Polished_Color_Lighten.lighten;
 let opacify = Polished_Color_Opacify.opacify;
 let readable = Polished_Color_Readable.readable;
 let transparentize = Polished_Color_Transparentize.transparentize;
+let mix = Polished_Color_Mix.mix;
+let shade = Polished_Color_Shade.shade;
+let tint = Polished_Color_Tint.tint;
+
```

Comment:
These could be in alphabetical order too.

Reply:
done